arches/app/models/resource.py

```diff
@@ -228,9 +228,11 @@ def save(self, *args, **kwargs):
         context = kwargs.pop("context", None)
         transaction_id = kwargs.pop("transaction_id", None)
         super(Resource, self).save(*args, **kwargs)
+        self.save_edit(user=user, edit_type="create", transaction_id=transaction_id)
```

Comment:
I think as the ticket author pointed out it's possible that moving this call to solve #9769 is not necessary if we do the filtering in #9768 as in this PR, but I still like the idea of doing it in case the another caller somewhere else in the application chooses not to filter in the same precise way.